localization/strings/en-GB/Resources.resw

```diff
@@ -181,7 +181,7 @@ Install using 'wsl.exe {} &lt;Distro&gt;'.
   <data name="MessageExportFailed" xml:space="preserve">
     <value>Exporting the distribution failed.</value>
   </data>
-  <data name="MessageFsUpdgadeNeeded" xml:space="preserve">
```

Comment:
Some project don't like changing localization keys. I'm happy to drop such changes.

Note that I've skipped touching non en- localizations, but I can certainly update the keys for the items I'm changing here if that's desired. It's a coin-toss and no matter which way I go, I'm usually wrong...

Reply:
In other words, I'm open to this change, it should just be done in the en-US version of the xaml file, and as a separate PR please.

Reply:
- #12957